Packs/CommonTypes/IncidentFields/incidentfield-User_Groups.json

```diff
@@ -0,0 +1,31 @@
+{
+    "id": "incident_usergroups",
+    "version": -1,
+    "modified": "2023-06-13T21:36:13.191337431Z",
+    "name": "User Groups",
+    "ownerOnly": false,
+    "cliName": "usergroups",
+    "type": "multiSelect",
```

Comment:
Just making sure - this is a dynamic list, not a static list, right?

Reply:
Yes